StochasticBarrierFunctions/src/StochasticBarrierFunctions.jl

```diff
@@ -56,6 +56,7 @@ include("barrier_algorithms/sum_of_squares_barrier.jl")
 # Submodules
 include("Plots/Plots.jl")
 include("Data/Data.jl")
+export load_probabilities, load_dynamics
```

Comment:
```suggestion
```
The purpose was to not pollute the namespace of `StochasticBarrierFunctions` so rather you should import these functions using `using StochasticBarrierFunctions.Data`.